code/modules/food_and_drinks/food/foods/pizza.dm

```diff
@@ -318,6 +318,12 @@
 		return
 	..()
 
+/obj/item/pizzabox/AltClick(mob/user)
+	..()
```

Comment:
You definitely want some sanity checks here, so people can't do wireless pizza opening
```suggestion
	..()
	if(length(boxes) || !Adjacent(user))
		return
```